### ISSUE TITLE
Show whether match is a superlike from messages page

### ISSUE THREAD

**Issue by mayeaux** [tinderjs/tinder-desktop#25]
There needs to be something added in the UI that indicates whether a user is a superlike or not 

My best bet is to add it with the 'Display Match Conversation Summary' details section, probably a little star icon if is_superlike is true for a given match

**Comment by SaturnusDJ** [comment]
Yeah indeed! To me it would be important to be always visible apart from 'Display Match Conversation Summary.'

Maybe like this:
Given superlike -> blue heart.
Received superlike -> red heart.
Both superliked -> purple heart.

**Comment by mayeaux** [comment]
Yup, I like it. 👍 I think even this will be an improvement over the actual client, because I don't know if you can tell who superliked who on the mobile client
